### PR TITLE
Added UTF-8 byte order mark for csv export

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
…ator

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Please review.

File: openslides/core/static/js/core/csv.js [hunk 1]
```diff
@@ -5,13 +5,17 @@
...
 
 .factory('CsvDownload', [
+    'Config',
     'FileSaver',
-    function (FileSaver) {
+    function (Config, FileSaver) {
         var utf8_BOM = '%EF%BB%BF';
         return function (contentRows, filename) {
+            var separator = Config.get('csv_separator').value;
+            console.log(separator);
```
Comment (emanuelschuetze) [inline openslides/core/static/js/core/csv.js]: please remove this line

File: openslides/core/config_variables.py [hunk 2]
```diff
@@ -105,6 +105,16 @@ def get_config_variables():
         group='General',
         subgroup='System')
 
+    # CSV
+
+    yield ConfigVariable(
+        name='csv_separator',
```
Comment (normanjaeckel) [inline openslides/core/config_variables.py]: Please rename this variable to `general_csv_separator`.

Comment (FinnStutzenstein) [inline openslides/core/config_variables.py]: Done.

**Comment by normanjaeckel** [comment]
Please add CHANGELOG notice. Then merge.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel @emanuelschuetze Added CHANGELOG entry

**Pull request merged**